### ISSUE TITLE
Issue with extrusion

### ISSUE THREAD

**Issue by Tpullmann** [xBimTeam/XbimGeometry#237]
Hi. 
I got an IFC which renders visually "correct" on other viewers like BimVision, but in XBim we see the reinforcement bars not extruded. Is the IFC too messed up or shoud XBim be more greedy to these kinds of issues?
[Reinforcement_issue.zip](https://github.com/xBimTeam/XbimGeometry/files/4030315/Reinforcement_issue.zip)

**Comment by andyward** [comment]
This seems to be OK in the very latest 5.1 Geometry (in the develop branch).

![image](https://user-images.githubusercontent.com/923699/71898602-a34bfe80-3151-11ea-809d-ba3b89d273a9.png)

**Comment by Tpullmann** [comment]
Yeah, that's the way it is supposed to look like, at least till 5.1.283 it was broken.
When do you expect the current branch to be released via nuget?

**Comment by ehtick** [comment]
It is broken for a while now, it just to be a setting.
For future updates these settings should not be removed, ifc generated from revit all have this problem. This is a bug from the revit exporter but they don't fix it.

Rebar is the worst :(

**Comment by andyward** [comment]
Shouldn't be long before Steve pushes to master,and we can make a nuget release.

If urgent you can get a build from the develop myget at https://www.myget.org/F/xbim-develop/api/v3/index.json

**Comment by ehtick** [comment]
Is there also a 5.x version?

**Comment by martin1cerny** [comment]
Yes, all our latest builds are there.
https://www.myget.org/feed/xbim-develop/package/nuget/Xbim.Geometry
